src/Component/Navbar/Navbar.jsx

```diff
@@ -1,67 +1,74 @@
-import React, { useState } from 'react';
-import './Navbar.css';
-import { Link } from 'react-router-dom';
-import { Drawer } from '../Drawer/Drawer';
-import { useAuth } from '../../hooks/context/AuthContext';
+import React, { useState } from 'react'
+import './Navbar.css'
+import { Link, useLocation } from 'react-router-dom'
+import { Drawer } from '../Drawer/Drawer'
+import { useAuth } from '../../hooks/context/AuthContext'
 
-const Navbar = () => {
-  const {userDetail, setuserDetail} = useAuth();
-  const {token,user} = userDetail;
+const Navbar = ({ searchInput, searchHandler }) => {
+  const { userDetail, setuserDetail } = useAuth()
+  const { token, user } = userDetail
   const [sideBar, setsideBar] = useState(true)
+  const location = useLocation()
 
   const logoutHandler = () => {
-    localStorage.removeItem("skyEncodedToken")
-    localStorage.removeItem("skyUser")
-    setuserDetail({token:"", user:{}}) 
+    localStorageNaNpxoveItem('skyEncodedToken')
+    localStorageNaNpxoveItem('skyUser')
```

Comment:
I think this is a typo. Should be `localStorage.removeItem` can you check into this?

Reply:
Yeah, fixed this thanks kartik.